app/js/backbone-views.js

```diff
@@ -1498,6 +1499,13 @@ var FileSaveView = Backbone.View.extend({
         delete properties.mapType; // already stored in sbgn file, no need to store in extension as property
         chiseInstance.saveAsSbgnml(filename, version, renderInfo, properties);
       }
+      else if(fileformat === "celldesigner") {
+        var celldesignerText = text;
+        var blob = new Blob([celldesignerText], {
```

Comment:
@gokberkkaraca do you really need `celldesignerText`? Provide `text` directly to `Blob`.